lib/src/sync.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
 
   getPage(type, page, options) {
     let params = {
-      per_page: 25,
+      per_page: 100,
```

Comment:
@demirgazetic I see that here is changed to `per_page: 100`, but at the `getAll`, it looks like it's not applied this change. Let me know if it was intended.

Reply:
@schabibi1  thank you for the feedback. Yes, you are right I was using per_page: 100 in the getPage method, but in getAll I was using 25. Everything should be 100, as the rate limit is that—source:  [Storyblok Documentation](https://www.storyblok.com/docs/api/content-delivery/v2/getting-started/pagination).

I will follow up with a new PR to update this.

Reply:
@demirgazetic Thanks for checking our docs for the rate limit! Let me know when you update the change for `per_page`.

Reply:
@schabibi1, it's updated, but not in getPage, it's in getAll:

Old value: `let lastPage = Math.ceil(initialResponse.total / 25);`
New value: `let lastPage = Math.ceil(initialResponse.total / 100);`

Reply:
Thank you, @demirgazetic !
Yes, `getAll` looks aligned now. It looks good to me, and I'll proceed with the approval process internally.